packages/augur-state/src/run.ts

```diff
@@ -25,5 +25,22 @@ export async function start() {
   return controller.run();
 }
 
-// This will need to change when we place augur-state in a worker
-start();
+const isNode =
+  typeof process !== 'undefined' &&
+  process.versions != null &&
+  process.versions.node != null;
+
+// Use this to see if you're running within a webworker
```

Comment:
Should this be removed?

Reply:
Yes, I left it in for debugging purposes.

If you node src/run.ts it all starts. 

The worker, if we go with a service worker, will want to be able to start and stop fetching dynamically. It's ultimately cruft until we sort out what we really want to happen.